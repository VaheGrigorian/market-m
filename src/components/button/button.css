@import "../../theme/colors.css";

.button {
  background-color: var(--errorRed);
}