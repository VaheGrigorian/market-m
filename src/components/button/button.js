import React from "react";

import "./button.css";

export const Button = ({ children }) => (
  <button type="button" styleName="button">
    {children}
  </button>
);
