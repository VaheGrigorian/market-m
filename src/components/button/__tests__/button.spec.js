import React from "react";
import { render } from "react-testing-library";
import { Button } from "../button";

describe("Button", () => {
  test("renders", () => {
    let container = render(<Button />);
    expect(container).toMatchSnapshot();
  });
});
