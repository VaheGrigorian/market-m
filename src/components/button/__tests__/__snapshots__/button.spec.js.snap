// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <button
        class="src-components-button-___button__button___lmpwR"
        type="button"
      />
    </div>
  </body>,
  "container": <div>
    <button
      class="src-components-button-___button__button___lmpwR"
      type="button"
    />
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
